scala3doc/src/dotty/dokka/compat.scala

```diff
@@ -57,3 +59,9 @@ extension [V] (map: JMap[SourceSetWrapper, V]):
 extension [V](jlist: JList[V]):
   def ++ (other: JList[V]): JList[V] =
     Stream.of(jlist, other).flatMap(_.stream).collect(Collectors.toList())
+
+extension (m: Any):
```

Comment:
I do not think that this is a good idea to have that extension on topLevel (see comments below)